.github/workflows/run-tests.yml

```diff
@@ -17,3 +17,15 @@ jobs:
           key: ${{ runner.os }}-sbt-cache-${{ hashFiles('**/build.sbt') }}-${{ hashFiles('**/plugins.sbt') }}-${{ hashFiles('**/build.properties') }}
       - name: Compile and validate
         run: sbt +validate
+
+  merge:
+    name: Merge
+    needs:
+      - test
```

Comment:
Every PR that successfully executed tests gets merged?
Is that a good idea for a public repository?

Reply:
Nope, only the ones by `moia-pipeline`:
https://github.com/ridedott/merge-me-action#enable-auto-merge-for-a-different-bot

Reply:
Cool - thanks